csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -31,6 +31,19 @@ public enum ExecutionMode
         ORT_PARALLEL = 1,
     }
 
+    /// <summary>
+    /// Language projection property for telemetry event for tracking the source usage of ONNXRUNTIME
+    /// </summary>
+    public enum OrtProgrammingProjection
+    {
+        ORT_PROJECTION_CROSSPLAT_C = 0,
+        ORT_PROJECTION_CPLUSPLUS,
+        ORT_PROJECTION_CROSSPLAT_CSHARP,
+        ORT_PROJECTION_CROSSPLAT_PYTHON,
+        ORT_PROJECTION_CROSSPLAT_JAVA,
+        ORT_PROJECTION_WINML,
```

Comment:
did we decide it would be "WINML" or "WINRT"?

Reply:
We didn't decide, but we prefered not to use WINRT, but something WINML, so I used WINML.

Reply:
i prefer winml.  it is the proper name of the winml projection (which happens to be winrt)